public/components/add_layer_panel/add_layer_panel.tsx

```diff
@@ -108,7 +110,7 @@ export const AddLayerPanel = ({
           fill
           onClick={showModal}
           aria-label="Add layer"
-          isDisabled={IsLayerConfigVisible}
+          isDisabled={IsLayerConfigVisible || layerCount >= MAX_LAYER_LIMIT}
```

Comment:
when the add layer button is disabled, I am not able to see the tool-tip which provides info around why this button is disabled. If it is not added, please add the tool-tip, otherwise please refer the code here for the tool tip.

Reply:
Added. I was waiting for text review .